doc/enhanced-dev-preview-installation-steps.md

```diff
@@ -17,7 +17,23 @@ From the terminal, login into ACM hub cluster environment using login command. Y
 oc login --token=*** --server=***
 ```
 
-### Step 2: Deploy Policy 
+### Step 2: Setup ManagedClusterSetBinding
+We need a **ManagedClusterSet** and a **ManagedClusterSetBinding** to ensure that the `Placement` can select clusters. If you already have these, you can skip this step (you can use the existing ManagedClusterSet in **Step 4**). Otherwise, you can create a ManagedClusterSet and bind it to the `default` namespace using the command below.    
+
+```
+oc apply -f data-assets/rs-managedclustersetbinding.yaml
+```
+
+**Assign Clusters to the ManagedClusterSet**: You can assign multiple clusters to the created ManagedClusterSet either through the ACM UI by navigating to `/multicloud/infrastructure/clusters/sets/details/rs-cluster-set/manage-resources` after the host URL, or by using the sample command below. 
```

Comment:
I noticed clusters were automatically selected after applying the ManagedClusterSet and ManagedClusterSetBinding. Do we need the following instructions? Maybe we can keep this part as optional?